src/ecs-additions/fetch_and_run.sh

```diff
@@ -16,6 +16,7 @@
 # See below for usage instructions.
 
 #PATH="/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/aws-cli/v2/current/dist
```

Comment:
This will work for Cromwell and Step Functions, but need to add something similar to the generated `nextflow.config` file:
https://github.com/aws-samples/aws-genomics-workflows/blob/f2de6e29354a49e5cd2ed8225daf5333e3b36c54/src/containers/nextflow/nextflow.aws.sh#L43-L48

See also:
https://www.nextflow.io/docs/latest/config.html#scope-env